common/flags/Flags.java

```diff
@@ -91,10 +90,11 @@ public static void printUsage() {
 
   @VisibleForTesting
   static String getFlagValue(String name) {
-    FlagData flagData = instance().getFlag(name);
+    FlagData flagData = getFlag(name);
     if (flagData == null) {
       throw new IllegalArgumentException(
           String.format(
+              // TODO flagData is always null. Think of a more informative message
               "Flag data '%s' is null; did you forget to call FlagData.parse()?", flagData));
```

Comment:
s/flagData/name (in `String.format()`).
Also, let's also mention forgetting to add the package:
Let's change:
did you forget to call FlagData.parse()?
To:
did you forget to call FlagData.parse() or add the package for the flag?